src/main/scala/is/hail/asm4s/FunctionBuilder.scala

```diff
@@ -45,6 +45,9 @@ object FunctionBuilder {
   def functionBuilder[A: TypeInfo, B: TypeInfo, R: TypeInfo]: FunctionBuilder[AsmFunction2[A, B, R]] =
     new FunctionBuilder(Array(GenericTypeInfo[A], GenericTypeInfo[B]), GenericTypeInfo[R])
 
+  def functionBuilder[A: TypeInfo, B: TypeInfo, R: TypeInfo](pName: String): FunctionBuilder[AsmFunction2[A, B, R]] =
+    new FunctionBuilder(Array(GenericTypeInfo[A], GenericTypeInfo[B]), GenericTypeInfo[R], packageName = pName)
```

Comment:
These classes are intended as temporary classes. Is there a particular reason the package name needs to be customizable?

Reply:
nope. forgot to take that out while I was poking at something.